packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -142,12 +143,13 @@ type GetCallCompositePageFunction = ((
  */
 export const getCallCompositePage: GetCallCompositePageFunction = (
   call,
-  previousCall,
+  previousCall?,
```

Comment:
functions should avoid many optional positional params, instead this should be an object with named params